app/Exceptions/Handler.php

```diff
@@ -97,7 +97,9 @@ protected function unauthenticated($request, AuthenticationException $exception)
     protected function renderHttpException(HttpExceptionInterface $e)
     {
         if (! view()->exists("errors.{$e->getStatusCode()}")) {
-            return response()->view('errors.default', ['exception' => $e, 'hide_message' => (! Auth::check() || Auth::user()?->canNot('finadmin'))], 500, $e->getHeaders());
+            $maySeeError = App()->env('local') | Auth::check() && Auth::user()->can('finadmin');
```

Comment:
maybe use `||` and shouldn't `finadmin` just be `admin` or `sysadmin`

Reply:
No, due to a bug in our financial system the treasurer should be able to also see the errors until it is fixed. See last minutes